File: app/api/meeting.py
import requests
import json

API_KEY = "talgiving_default_key"
AUDIO_ROOMS_URL = "http://localhost:4000/api/v1/meeting"

headers = {
    "authorization": API_KEY,
    "Content-Type": "application/json",
}

response = requests.post(
    AUDIO_ROOMS_URL,
    headers=headers
)

print("Status code:", response.status_code)
data = json.loads(response.text)
print("meeting:", data["meeting"])

File: app/api/meeting.js
'use strict';

const fetch = require('node-fetch');

const API_KEY = 'talgiving_default_key';
const AUDIO_ROOMS_URL = 'http://localhost:4000/api/v1/meeting';

function getResponse() {
    return fetch(AUDIO_ROOMS_URL, {
        method: 'POST',
        headers: {
            authorization: API_KEY,
            'Content-Type': 'application/json',
        },
    });
}

getResponse().then(async (res) => {
    console.log('Status code:', res.status);
    const data = await res.json();
    console.log('meeting:', data.meeting);
});
